ledger-tool/src/main.rs

```diff
@@ -3269,6 +3283,57 @@ fn main() {
                 }
             }
         }
+        ("stats", Some(arg_matches)) => {
+            let process_options = ProcessOptions {
+                dev_halt_at_slot: value_t!(arg_matches, "halt_at_slot", Slot).ok(),
+                new_hard_forks: hardforks_of(arg_matches, "hard_forks"),
+                bpf_jit: !matches.is_present("no_bpf_jit"),
+                accounts_db_caching_enabled: !arg_matches.is_present("no_accounts_db_caching"),
+                limit_load_slot_count_from_snapshot: value_t!(
+                    arg_matches,
+                    "limit_load_slot_count_from_snapshot",
+                    usize
+                )
+                .ok(),
+                accounts_db_config: None,
+                verify_index: arg_matches.is_present("verify_accounts_index"),
+                allow_dead_slots: arg_matches.is_present("allow_dead_slots"),
+                accounts_db_test_hash_calculation: arg_matches
+                    .is_present("accounts_db_test_hash_calculation"),
+                ..ProcessOptions::default()
+            };
```

Comment:
It some point it'll be worth building a helper function for the `ProcessOptions` handling that's been copied at least a couple times now (as well as the clap `SubCommand` `.arg()` duplication).  I don't feel that must happen in this PR though!